### DISCUSSION TITLE
[Q]: Workflow does not have 'workflow_dispatch' trigger error

### DISCUSSION THREAD

**Discussion by janc-pxl** [ObsidianPublisher/obsidian-github-publisher#100] [Q&A — Template] [answered]
### Issue validation

- [X] My issue is not a question relative to [Material Mkdocs](https://squidfunk.github.io/mkdocs-material/) nor [Mkdocs](https://www.mkdocs.org/) (*[You can open a discussion to get more help!](https://github.com/squidfunk/mkdocs-material/discussions)*)
- [X] I checked the issue to prevent duplicate
- [X] I checked my configurations files and the documentation

### Workflow

Obsidian plugin 💠

### Describe the bug

```
plugin:obsidian-mkdocs-publisher:2236 HttpError: Workflow does not have 'workflow_dispatch' trigger
    at eval (plugin:obsidian-mkdocs-publisher:1037:21)
    at Generator.next (<anonymous>)
    at fulfilled (plugin:obsidian-mkdocs-publisher:47:24)
```

### How to reproduce ?

Do a Share active file

### Configuration

```
{
  "githubRepo": "pxl-projectmanagement-d",
  "githubName": "janc-pxl",
  "GhToken": "******",
  "shareKey": "share",
  "ExcludedFolder": "",
  "fileMenu": false,
  "editorMenu": false,
  "downloadedFolder": "fixed",
  "folderDefaultName": "docs",
  "yamlFolderKey": "",
  "rootFolder": "",
  "workflowName": "ci.yml",
  "transferEmbedded": true,
  "defaultImageFolder": "",
  "autoCleanUp": false,
  "autoCleanUpExcluded": "",
  "folderNote": false,
  "convertWikiLinks": false,
  "convertForGithub": false,
  "subFolder": ""
}
```

### Relevant log output

```shell
plugin:obsidian-mkdocs-publisher:2236 HttpError: Workflow does not have 'workflow_dispatch' trigger
    at eval (plugin:obsidian-mkdocs-publisher:1037:21)
    at Generator.next (<anonymous>)
    at fulfilled (plugin:obsidian-mkdocs-publisher:47:24)
```
```


### OS

MacOS

### Anything else?

_No response_

**Comment by Mara-Li** [comment]
You have a workflow named `ci.yml` that don't have a workflow dispatch event. The plugin can't activate it either ! 
Did you check your workflow file ? If it's a push event you can safely remove it from the configuration and activation. 

Note : if you use the plugin with the last version of the template you don't need the conversion and the workflow. You can safely delete the `ci.yml` file.

**Comment by janc-pxl** [comment, marked as answer]
Deleted the ci workflow did the trick
